tzrec/modules/intervention.py

```diff
@@ -0,0 +1,36 @@
+import torch
+from torch import nn
+
+class RotateLayer(nn.Module):
```

Comment:
add unit test for these module in intervention_test.py

Reply:
Added intervention_test.py in modules/